jobbergate-api/jobbergate_api/tests/apps/job_submissions/test_routers.py

```diff
@@ -80,6 +86,10 @@ async def test_create_job_submission__with_client_id_in_token(
     assert job_submission.created_at in window
     assert job_submission.updated_at in window
 
+    assert job_submission.execution_parameters is not None
+    assert job_submission.execution_parameters.name == "job-submission-name"
+    assert job_submission.execution_parameters.comment == "I am a comment"
```

Comment:
it is missing to check in the database if the data is indeed there. If you don't check that, it is possible that the data isn't there but the response is the one you expect.

Reply:
It was addressed in https://github.com/omnivector-solutions/jobbergate/pull/193#discussion_r1041057788